pavelib/quality.py

```diff
@@ -315,6 +317,64 @@ def run_eslint(options):
         )
 
 
+def _get_stylelint_violations():
+    """
+    Returns the number of Stylelint violations.
+    """
+    stylelint_report_dir = (Env.REPORT_DIR / "stylelint")
+    stylelint_report = stylelint_report_dir / "stylelint.report"
+    _prepare_report_dir(stylelint_report_dir)
+    formatter = 'node_modules/stylelint-formatter-pretty'
+
+    sh(
+        "stylelint **/*.scss --custom-formatter={formatter} | tee {stylelint_report}".format(
+            formatter=formatter,
+            stylelint_report=stylelint_report,
+        ),
+        ignore_error=True
+    )
+
+    try:
+        return int(_get_count_from_last_line(stylelint_report, "eslint"))
+    except TypeError:
+        raise BuildFailure(
+            "Error. Number of eslint violations could not be found in {eslint_report}".format(
```

Comment:
3x `eslint` => `stylelint`.  Search throughout this function.

Reply:
Fixed